Formula/x11vnc.rb

```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```

Comment:
Can you please revert this whitespace change as already pointed out by my fellow Homebrew maintainer in #3628?